pioinstaller/__main__.py

```diff
@@ -13,18 +13,41 @@
 # limitations under the License.
 
 import os
+import subprocess
 import sys
 
 import click
 
 from pioinstaller import __title__, __version__
 from pioinstaller.pack import packer
+from pioinstaller.python import check as python_check
+from pioinstaller.util import get_pythonexe_path
 
 
-@click.group()
+@click.group(name="main", invoke_without_command=True)
 @click.version_option(__version__, prog_name=__title__)
-def cli():
-    pass
+@click.pass_context
+def cli(ctx):
+    if not ctx.invoked_subcommand:
+        exenames = ["python3", "python", "python2"]
+        if sys.platform.lower().startswith("win"):
```

Comment:
I see you duplicate the same code in multiple places. Plase create constant in `util`
```
IS_WINDOWS = sys.platform.lower().startswith("win")
```